benchmark/factory/legged-robots.hpp

```diff
@@ -115,11 +128,11 @@ void build_biped_action_models(boost::shared_ptr<crocoddyl::ActionModelAbstractT
       boost::make_shared<DifferentialActionModelContactFwdDynamics>(state, actuation, contact_models,
                                                                     terminalCostModel);
 
-  runningModel = boost::make_shared<IntegratedActionModelEuler>(runningDAM, Scalar(1e-3));
-  terminalModel = boost::make_shared<IntegratedActionModelEuler>(terminalDAM, Scalar(1e-3));
+  runningModel = boost::make_shared<IntegratedActionModelEuler>(runningDAM, Scalar(5e-3));
```

Comment:
These `dt` are sort of big, they might trigger a few more iterations. 
I suggest you to keep it as before, so we encourage to bench a single backward/forward passes in the solvers.

Reply:
This is the standard time step used in simulations. Ideally, we should be benching with even higher `dt`, to show the preview we can obtain. But I've kept it as this since it is widely used.